packages/aws-cdk/package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "cdk-lint",
     "pkglint": "pkglint -f",
     "test": "cdk-test",
-    "integ": "jest --testMatch '**/?(*.)+(integ-test).ts'",
+    "integ": "jest --testMatch '**/?(*.)+(integ-test).js'",
```

Comment:
can we call these by another name so they are not confused with our normal integ tests?

Reply:
Sure. How about "clinteg test" ?